### ISSUE TITLE
Registraction part is missing!

### ISSUE THREAD

**Issue by esQmo** [probelalkhan/android-login-signup-tutorial#4]
Is it just me or the signup part is missiong?

**Comment by whmarti** [comment]
Yes, strangely, the code shown in the video is different from the one available for download. And the one that is here, is not working ..
is it updating or what happened?
